framework/scenario/tests/contract_without_macros.rs

```diff
@@ -102,64 +105,63 @@ mod module_1 {
         }
     }
 
-    pub trait ProxyTrait: multiversx_sc::contract_base::ProxyObjBase + Sized {
+    pub trait ProxyTrait: ProxyObjBase + Sized {
```

Comment:
Leave the fully qualified version in place, this is how it gets generated by the macros